ui/src/components/Organization/List.tsx

```diff
@@ -9,17 +9,33 @@ interface Props {
   organizations: Organization[];
 }
 
+const defaultBand = (brand: string, name: string) => {
```

Comment:
I believe you wanted to name it `defaultBrand`? If yes, there is a typo in the name (it says `Band`).